db/README.md

```diff
@@ -0,0 +1,15 @@
+# asset-wizard db
+
+## Getting Started
+
+### How to run database in local
+
+Run `docker build -t assetwizard_db .` then it will build PostgreSQL container with SQL files in this directory. After that, run following command then you can connect to PostgreSQL via local 5432 port.
+
+```bash
+docker run -it \
+  -e POSTGRES_DB=asset-wizard \
+  -e POSTGRES_PASSWORD=ayush \
```

Comment:
I am not sure about hard coding ayush here. But technically properties file also has this password hard-coded.
I will change this later, when i remove that hard-coding from properties file